programmers/난이도별/level01.크레인_인형뽑기_게임/HyeonJeong.py

```diff
@@ -0,0 +1,19 @@
+def solution(board, moves):
+    answer = 0
+    slist = []
+    length = len(board)
+    for m in moves:
+        if board[length-1][m-1] == 0:
+            continue
+        for i in range(length):
+            if board[i][m - 1] != 0:
+                if len(slist) > 0 and slist[-1] == board[i][m - 1]:
+                    answer += 2
+                    slist.pop(-1)
+                else:
+                    slist.append(board[i][m - 1])
+                board[i][m - 1] = 0
+                break
```

Comment:
```suggestion
            if board[i][m - 1] == 0: continue
            if len(slist) > 0 and slist[-1] == board[i][m - 1]:
                answer += 2
                slist.pop(-1)
            else:
                slist.append(board[i][m - 1])
            board[i][m - 1] = 0
            break
```
이 부분도 이렇게 continue를 사용하면 좋을 것 같아요!